src/cdk/testing/tests/webdriver.e2e.spec.ts

```diff
@@ -20,11 +24,32 @@ const webTestMetadata: WebTestMetadata =
     require(runfiles.resolve(process.env['WEB_TEST_METADATA']));
 const port = process.env['E2E_APP_PORT'];
 
-describe('Webdriver test', () => {
-  let wd: WebDriver;
+// Kagekiri is available globally in the browser. We declare it here so we can use it in the
+// browser-side script passed to `by.js`.
+// TODO(mmalerba): Replace with type-only import once TS 3.8 is available, see:
```

Comment:
Not strictly necessary since this was in the protractor environment, but current TS is 4.2. Now the limiting factor for type-only imports is whether we can use them in Google (I'm actually not sure what the rule is for third_party code).

Reply:
Yeah I don't think this comment is even right. A type only import will just complain that I'm using it as a value instead of a type. I'll remove it here and in the protractor one.